acceptance/extension-logging-fluentd/package.json

```diff
@@ -38,5 +38,8 @@
     "type": "git",
     "url": "https://github.com/strongloop/loopback-next.git",
     "directory": "acceptance/extension-logging-fluentd"
+  },
+  "dependencies": {
+    "tslib": "^1.10.0"
```

Comment:
Nitpick: for `acceptance/*` packages, we only declare `devDependencies` as they are only for testing. Do we want to move `tslib` to `devDependencies` for these packages?

Reply:
@raymondfeng Updated 1st commit to move `tslib` for `acceptance/` packages to `devDependency`.

Also, the 4th commit (update package locks) was merged to the 1st commit to keep the git history clean.